custom_components/tuya_local/devices/plikc_neve_x_thermostat.yaml

```diff
@@ -0,0 +1,147 @@
+name: Termostato
+products:
+  - id: 5kwgvcaqzzkzrftu
+    name: Plikc Neve X W
+primary_entity:
+  entity: climate
+  translation_key: thermostat
+  dps:
+    - id: 1
+      type: boolean
+      name: hvac_mode
+      mapping:
+        - dps_val: false
+          value: "off"
+        - dps_val: true
+          value: heat
+    - id: 2
+      type: string
+      name: preset_mode
+      mapping:
+        - dps_val: auto
+          value: home
+        - dps_val: manual
+          value: boost
+        - dps_val: temporary
+          value: boost
+          hidden: true
+        - dps_val: holiday
+          value: away
+          constraint: Durata vacanza
+          conditions:
+            - dps_val: 1
```

Comment:
This seems incomplete. It also references a non-existing dp

Reply:
What I wanted to do was set the dp 33 (vacation duration) to 1, as the thermostat doesn't go in vacation mode if a value is not pushed to the vacation duration